config.go

```diff
@@ -6,16 +6,19 @@ import (
 	"os"
 
 	"github.com/BurntSushi/toml"
+	"github.com/nolanlum/tanya/irc"
 	"github.com/nolanlum/tanya/token"
 )
 
 type slack struct {
-	Token string
+	Token  string
+	UserID string
```

Comment:
Does this need to be in the config file? Can we retrieve information about the current user when we auth?

Reply:
szi discovered that the user/team ID are sent upon successful RTM connection so we are refactoring to burst all the relevant data at the beginning of the connection

Reply:
but on a side note the token you get from `auth.login` does not have the right scopes to get the user's identity.

Reply:
by "refactoring to burst all the relevant data" do you mean that's happening here or in a future change? (the latter, right?)

Reply:
it happened here because it didn't make sense to keep this config'd UserID nonsense